packages/core/src/modules/credentials/protocol/revocation-notification/services/RevocationNotificationService.ts

```diff
@@ -51,7 +51,21 @@ export class RevocationNotificationService {
     comment?: string
   ) {
     // TODO: can we extract support for this revocation notification handler to the anoncreds module?
-    const query = { anonCredsRevocationRegistryId, anonCredsCredentialRevocationId, connectionId: connection.id }
+    // Search for the revocation registry in both qualified and unqualified forms
+    const query = {
```

Comment:
Do we already store it in both formats? We MIGHT have to query the credential itself first and based on that find the exchange record (as we definitely store both qualified and unqualified on the w3c record)

Reply:
I think the idea in 0.5 was to store both in the w3c record, but up to now it is doing so only in case the issuer is a qualified did:indy, which is not the case in BCGov demo for example, so unqualified form is not being recorded there.

But something I'm thinking while inspectic the code is the following: `LegacyIndyCredentialFormatService` is supposed to work only with unqualified identifiers, right? In that case, to fix this particular issue shouldn't it suffice to simply store `revocationRegistryId` tag and metadata in unqualified format?

Reply:
> But something I'm thinking while inspectic the code is the following: LegacyIndyCredentialFormatService is supposed to work only with unqualified identifiers, right? In that case, to fix this particular issue shouldn't it suffice to simply store anonCredsRevocationRegistryId tag and metadata in unqualified format?

I think for the migration it's easier if we can store as much as possible as qualified identifiers. It's very simple to go from qualified -> unqualified, not the other way around. In the future it would be nice if you could send a revocation notification with a qualified identifier for a credential that was issued as unqualified.